app/src/org/commcare/utils/MediaUtil.java

```diff
@@ -370,21 +371,82 @@ private static Bitmap scaleDownToTargetOrContainer(String imageFilepath,
         int approximateScaleDownFactor = getApproxScaleDownFactor(newWidth, originalWidth);
         Bitmap b = inflateImageSafe(imageFilepath, approximateScaleDownFactor).first;
 
+        int orientation = ExifInterface.ORIENTATION_NORMAL;
+        try {
+            ExifInterface exif = new ExifInterface(imageFilepath);
+            orientation = exif.getAttributeInt(ExifInterface.TAG_ORIENTATION, ExifInterface.ORIENTATION_NORMAL);
+        } catch (IOException e) {
+            Logger.exception("Unable to read image file from disk: ", e);
+        } catch (Exception e) {
+            Logger.exception("Unable to read exif data from image file: ", e);
+        }

```

Comment:
You can remove the `catch (IOException e) {` from here and just have a generic catch as we are not handling those 2 very differently.